test/api_test.rb

```diff
@@ -269,99 +278,74 @@ class WorkerWithTags
     it "enumerates jobs in descending score order" do
       # We need to enqueue more than 50 items, which is the page size when retrieving
       # from Redis to ensure everything is sorted: the pages and the items withing them.
-      51.times { ApiWorker.perform_in(100, 1, "foo") }
+      51.times { ApiJob.perform_in(100, 1, "foo") }
 
       set = Sidekiq::ScheduledSet.new.to_a
 
       assert_equal set.sort_by { |job| -job.score }, set
     end
 
     it "has no enqueued_at time for jobs enqueued in the future" do
-      job_id = ApiWorker.perform_in(100, 1, "foo")
+      job_id = ApiJob.perform_in(100, 1, "foo")
       job = Sidekiq::ScheduledSet.new.find_job(job_id)
       assert_nil job.enqueued_at
     end
 
-    it "unwraps delayed jobs" do
-      Sidekiq::DelayExtensions.enable_delay!
-      Sidekiq::Queue.delay.foo(1, 2, 3)
-      q = Sidekiq::Queue.new
-      x = q.first
-      assert_equal "Sidekiq::Queue.foo", x.display_class
-      assert_equal [1, 2, 3], x.display_args
-    end
-
-    it "handles previous (raw Array) error_backtrace format" do
-      add_retry
-      job = Sidekiq::RetrySet.new.first
-      assert_equal ["line1", "line2"], job.error_backtrace
-    end
-
-    it "handles previous (marshalled Array) error_backtrace format" do
```

Comment:
base64 related tests like this were removed from sidekiq and it fails here so, I'm fine with it.